vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/FormulaFormatter.java

```diff
@@ -280,6 +280,7 @@ protected DecimalFormat getDecimalFormat(Locale locale) {
         DecimalFormat instance = (DecimalFormat) DecimalFormat
                 .getInstance(locale);
         instance.setGroupingUsed(false);
+        instance.setMaximumFractionDigits(15);
```

Comment:
If I understand correctly, this setting should match the number of fraction digits specified in `EXCEL_FORMULA_BAR_DECIMAL_FORMAT`. However, that format allows for 16 digits, not 15.

https://github.com/vaadin/flow-components/blob/0a86fa9cfaaf0d4cfd6517ca4489f12537cdae8b/vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/CellValueManager.java#L81

Reply:
Good catch. Aligned the number of decimal digits with the one used by the `CellValueManager` class.